package.json

```diff
@@ -28,7 +28,7 @@
     "matrix_lib_main": "./lib/index.ts",
     "matrix_lib_typings": "./lib/index.d.ts",
     "matrix_i18n_extra_translation_funcs": [
-        "newTranslatableError"
+        "UserFriendlyError"
```

Comment:
@kerryarchibald This PR requires a `element-web-app-team` approval which you're part of. I just want to make sure this is still on your list even after @dbkr approved?

Reply:
Thanks for the review pass @kerryarchibald ✅